lib/node_modules/@stdlib/lapack/base/dlange/lib/dlange.js

```diff
@@ -0,0 +1,72 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isLayout = require( '@stdlib/blas/base/assert/is-layout' );
+var format = require( '@stdlib/string/format' );
+var base = require( './base.js' );
+
+
+// MAIN //
+
+/**
+* Returns the value of one norm, or the Forbenius norm, or the infinity norm, or the element of largest absolute value of a real matrix A.
+*
+* @param {string} order - storage layout
+* @param {string} norm - specifies the value to be returned
+* @param {NonNegativeInteger} M - number of rows of `A`
+* @param {NonNegativeInteger} N - number of columns of `A`
+* @param {Float64Array} A - input array
+* @param {PositiveInteger} LDA - stride of the first dimension of `A` (a.k.a., leading dimension of the matrix `A`)
+* @param {Float64Array} work - workspace array
+* @throws {TypeError} first argument must be a valid order
+* @returns {number} matrix norm
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
+* var work = new Float64Array( 2 );
+*
+* var out = dlange( 'row-major', 'M', 2, 2, A, 2, work );
+* // out => 4.0
```

Comment:
```suggestion
* // returns 4.0
```

Reply:
I had this initially but it threw me a weird lint error. I'll check it again.

Reply:
I see, I did the following

```javascript
* dlange( 'row-major', 'M', 2, 2, A, 2, 1, 0, work, 1, 0 );
* // returns 4.0
```